functions-app/report-psychometrician-create-download/service/psychometrician-report.service.js

```diff
@@ -104,11 +106,17 @@ const psychometricianReportService = {
       throw error
     }
 
-    const arStat = await fs.stat(anomalyReportFilename)
-    this.logger.verbose(`${functionName}: anomaly report size: ${Math.round(arStat.size / 1024 / 1024)} MB`)
+    try {
+      const arStat = await fs.stat(anomalyReportFilename)
+      this.logger.verbose(`${functionName}: anomaly report size: ${Math.round(arStat.size / 1024 / 1024)} MB`)
+      filesToZip.push(anomalyReportFilename)
+    } catch (error) {
+      // Anomaly report may not have been produced if there weren't any anomalies
+      this.logger.info(`${functionName}: failed to stat ${anomalyReportFilename}: ${error.message}`)
+    }
 
     const zipfileName = `pupil-check-data-${moment().format('YYYY-MM-DD HHmm')}.zip`
```

Comment:
@jon-shipley non-blocker: shouldn't that be `zipFilename` ?

Reply:
Yes, good spot. 👏 Now fixed.